test/ServerComparison.FunctionalTests/NtlmAuthenticationTest.cs

```diff
@@ -70,66 +71,45 @@ public NtlmAuthenticationTests(ITestOutputHelper output) : base(output)
                         Assert.Equal(HttpStatusCode.Unauthorized, response.StatusCode);
                         Assert.Contains("NTLM", response.Headers.WwwAuthenticate.ToString());
                         Assert.Contains("Negotiate", response.Headers.WwwAuthenticate.ToString());
-
-                        logger.LogInformation("Testing /RestrictedNTLM");
-                        response = await httpClient.GetAsync("/RestrictedNTLM");
-                        responseText = await response.Content.ReadAsStringAsync();
-                        Assert.Equal(HttpStatusCode.Unauthorized, response.StatusCode);
-                        Assert.Contains("NTLM", response.Headers.WwwAuthenticate.ToString());
-                        // Note IIS can't restrict a challenge to a specific auth type, the native auth modules always add themselves.
-                        // However WebListener can.
-                        if (serverType == ServerType.WebListener)
-                        {
-                            Assert.DoesNotContain("Negotiate", response.Headers.WwwAuthenticate.ToString());
-                        }
-                        else if (serverType == ServerType.IISExpress)
-                        {
-                            Assert.Contains("Negotiate", response.Headers.WwwAuthenticate.ToString());
-                        }
+                        */
 
                         logger.LogInformation("Testing /Forbidden");
                         response = await httpClient.GetAsync("/Forbidden");
                         Assert.Equal(HttpStatusCode.Forbidden, response.StatusCode);
-                        */
+
                         logger.LogInformation("Enabling Default Credentials");
 
                         // Change the http client to one that uses default credentials
                         httpClient = deploymentResult.CreateHttpClient(new HttpClientHandler() { UseDefaultCredentials = true });
 
-                        logger.LogInformation("Testing /AutoForbid");
```

Comment:
No point in changing this, you're going to remove this behavior soon anyways right?
It was named after the ends, not the means.